tests/libvmi/factory.go

```diff
@@ -88,7 +89,7 @@ func NewAlpineWithTestTooling(opts ...Option) *kvirtv1.VirtualMachineInstance {
 }
 
 func cirrosMemory() string {
-	if checks.IsARM64(testsuite.Arch) {
+	if checks.IsARM64(libnode.GetArch()) {
```

Comment:
oh, `libvmi` already had a dependency on `testsuite`. I think that was *my* fault, long ago. Depending on `libnode` is better, but not great. I wonder if (on another PR) we can have the caller supply the architecture somehow?

```go
func init() {
  libvmi.RegisterDefaultOption(withTestAnnotation)
  libvmi.SetArch(libnode.GetArch)
}
```